tests/Spice86.Tests/CfgGraphDumper.cs

```diff
@@ -22,7 +20,8 @@ public List<string> ToAssemblyListing(Machine machine) {
     }
 
     private List<ICfgNode> DumpInOrder(Machine machine) {
```

Comment:
Don't reference the Machine anywhere, it's not to be used by the emulator.

It's just there as a convenience for C# emulated program code overrides, and is a legacy object.

Reply:
It was like that before and out of scope for this MR.